GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/rd170engine.cfg

```diff
@@ -0,0 +1,36 @@
+//  ==================================================
+//  RD-170 engine plume setup.
+//  ==================================================
+
+@PART[rd170_Energia]:FOR[RealPlume]:NEEDS[SmokeScreen]
```

Comment:
BEFORE not FOR
Also plume-dedicated patches should go in the plumes section rather than the mod-specific section, if possible.